lib/diameter/examples/code/GNUmakefile

```diff
@@ -25,8 +25,14 @@ MODULES   = $(EXAMPLES) $(EXAMPLES:%=%_cb)
 
 BEAM = $(MODULES:%=%.beam)
 
+ifeq ($(ERL_DETERMINISTIC),yes)
```

Comment:
You can remove any changes from this file as it is only used to build example code and the ERL_DETERMINISTIC will never be set there.